<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebCam</class>
 <widget class="QWidget" name="WebCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>497</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Видео</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Video must be here</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Опции работы и сохранение</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Сохранить</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Изображение</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="1">
              <widget class="QRadioButton" name="rb_jpg">
               <property name="text">
                <string>jpg</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QRadioButton" name="rb_png">
               <property name="text">
                <string>png</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rb_bmp">
               <property name="text">
                <string>bmp</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" rowspan="2" colspan="3">
              <widget class="QPushButton" name="pushButton_changeSaveFilePath">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Путь сохранения изображения</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Сохранить изображение</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Видео запись</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Кодек</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="comboBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="currentText">
                <string>H.264</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>H.264</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Motion JPEG</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MPEG-4</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_writeVideo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Записать видео</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_stopVideo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Завершить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Опции обработки и вывода изображения</string>
         </property>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>221</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Вывод обработки в  отдельном окне</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Настройки камеры</string>
         </property>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>111</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title">
           <string>Разрешение</string>
          </property>
          <widget class="QRadioButton" name="rb_vga">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>640 x 480</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="rb_1080p">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>1080P</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
